robot/src/main/java/us/ilite/robot/Robot.java

```diff
@@ -49,22 +41,22 @@
 
 
     // Module declarations here
-    private Superstructure mSuperstructure = new Superstructure();
+    private CommandManager mAutonomousCommandManager = new CommandManager();
+    private CommandManager mTeleopCommandManager = new CommandManager();
     private DriveController mDriveController = new DriveController(new StrongholdProfile());
     private Drive mDrive = new Drive(mData, mDriveController);
     private Elevator mElevator = new Elevator(mData);
     private HatchFlower mHatchFlower = new HatchFlower();
 
-    private DriverInput mDriverInput = new DriverInput(mDrive, mElevator, mHatchFlower, mSuperstructure, mData);
+    private DriverInput mDriverInput = new DriverInput(mDrive, mElevator, mHatchFlower, mAutonomousCommandManager, mTeleopCommandManager, mData);
```

Comment:
Be careful with this merge, I've seen inputs to the constructor dropped in earlier merges.